ecs-init/docker/docker.go

```diff
@@ -244,9 +246,16 @@ func (c *Client) getContainerConfig() *godocker.Config {
 		env = append(env, envKey+"="+envValue)
 	}
 
+	lables, err := generateLabelMap(envVariables["ECS_CONTAINER_LABELS"])
```

Comment:
this triggers error when ECS_CONTAINER_LABELS is not set:
```
2019-10-14T18:35:37Z [ERROR] Failed to decode the container labels, skipping labels. Error: unexpected end of JSON input
```
Can you skip generateLabelMap when ECS_CONTAINER_LABELS is empty so that we don't have an error log by default?